ckanext/external_storage/authz.py

```diff
@@ -19,6 +19,8 @@ def check_object_permissions(id, dataset_id=None, organization_id=None):
     This wrap's ckanext-authz-service's default logic for checking resource
     by checking for global-prefix/dataset-uuid/* style object scopes.
     """
+    # support for resource_id/activity_id
+    id = id.split('/')[0]
```

Comment:
This is a bad consequence of a need to pass composite serialized id `resource_id/activity_id` which is a workaround put no to change `ckanext-authz-service` API.